_wormtablemodule.c

```diff
@@ -22,16 +22,25 @@
 #include <db.h>
 #include "halffloat.h"
 
+/* $VR$: Dealing with Windows... */
+/* the declspec allows compiling of a dll in Visual Studio
+   and simply renaming the file to .pyd for it to work as a 
+   setuptools-compiled module */
+
 #ifdef _WIN32
+#define DLLEXPORT
```

Comment:
I think this will need to be defined for non windows builds as well, right? Otherwise compile will fail below.